common/app/controllers/EmailSignupController.scala

```diff
@@ -42,6 +42,7 @@ object ListIds {
   val greenLight = 28
   val povertyMatters = 113
   val theLongRead = 3322
+  val weekendReading = 3743
```

Comment:
With regards for what they mean, in this file the listID is used to optionally identify an Editorial email, so that a theme/tone can be added to it (comment, feature, etc.).